### ISSUE TITLE
Compiling for tvOS is broken on stock install.

### ISSUE THREAD

**Issue by Diatrus** [theos/theos#395]
(Thanks for reporting an issue! Please make sure you click the link above to view the issue guidelines, then fill out the blanks below.)

What are the steps to reproduce this issue?
-------------------------------------------
1. Attempt to compile a tweak for AppleTV (tvOS).

What happens?
-------------
An error `bash: -x: command not found` meaning that theos is substituting a blank string for what should be the compiler.

What were you expecting to happen?
----------------------------------
Successful compile

Any logs, error output, etc?
----------------------------
(If it’s long, please paste to https://ghostbin.com/ and insert the link here.)
https://pastebin.com/6wvUiFhb

Any other comments?
-------------------
Replacing line 58 of darwin_head.mk with line 61 fixes the issue, but it isn't a true fix, more of a workaround. This means that theos isn't finding the AppleTV SDK, I suppose. This is on a stock system/stock theos install. I reinstalled theos 3 times, used 2 different versions of XCode/command line tools.

What versions of software are you using?
----------------------------------------
**Operating System:** maOS

**Toolchain Version:** N/A?

**SDK Version:** 10.1/12.2

**Comment by uroboro** [comment]
Please share your makefile. I have a hunch about what's going on but I need to see the `TARGET` you've set.

**Comment by Diatrus** [comment]
[Makefile.txt](https://github.com/theos/theos/files/3035835/Makefile.txt)
@uroboro

**Comment by uroboro** [comment]
Pasting for ease of reading:
```makefile
ARCHS = arm64
TARGET = appletv
export GO_EASY_ON_ME=1
export SDKVERSION=10.1
TARGET = appletv:clang:10.1:10.1
SYSROOT = $(THEOS)/sdks/AppleTVOS10.1.sdk
DEBUG = 0

THEOS_PACKAGE_DIR_NAME = debs
PACKAGE_VERSION = $(THEOS_PACKAGE_BASE_VERSION)

include $(THEOS)/makefiles/common.mk

SUBPROJECTS += AppSyncUnified
SUBPROJECTS += postinst
SUBPROJECTS += preinst
include $(THEOS_MAKE_PATH)/aggregate.mk

clean::
	@rm -f debs/*.deb
```
(You should fix: duplicate `TARGET`, not exporting `ARCHS` and `TARGET`, having an unnecessary `THEOS_PACKAGE_DIR_NAME`, `THEOS_PACKAGE_BASE_VERSION` is not set before `$(THEOS)/makefiles/common.mk` is included)

Yeah, we have an issue with how the AppleTV target is named. We use `appletv` on [$THEOS/makefiles/targets/Darwin/appletv.mk](https://github.com/theos/theos/blob/master/makefiles/targets/Darwin/appletv.mk) while Xcode has AppleTVOS:

![image](https://user-images.githubusercontent.com/1142354/55434855-d34dec00-5598-11e9-94e0-6296f1b851b1.png)

For future reference, it's consistent excluding macOS:
![image](https://user-images.githubusercontent.com/1142354/55435001-3c356400-5599-11e9-8dba-7b08bab2dd72.png)

**Comment by uroboro** [comment]
If this is the case, then https://github.com/theos/theos/blob/450b719aa7bfc4f89c9db8ede6d0538254e666a5/makefiles/targets/Darwin/appletv.mk#L5 should be changed to `appletvos` like the simulator has: https://github.com/theos/theos/blob/450b719aa7bfc4f89c9db8ede6d0538254e666a5/makefiles/targets/Darwin/appletv_simulator.mk#L5

**Comment by uroboro** [comment]
Should be fixed in 827684e

**Comment by Diatrus** [comment]
Fixed
